libsolc/CMakeLists.txt

```diff
@@ -1,8 +1,39 @@
+
+# CreateExportedFunctions(OUTPUT_VARIABLE Symbol1 Symbol2 ... SymbolN)
+macro(CreateExportedFunctions)
```

Comment:
I'd put the macro in some utils file in `cmake/`. Other than that, having this does seem like an improvement as long as it works correctly.